src/LaravelDebugbar.php

```diff
@@ -404,6 +405,15 @@ function ($query, $bindings = null, $time = null, $connectionName = null) use ($
             }
         }
 
+        if ($this->shouldCollect('models', false)) {
```

Comment:
The model collector only runs if the flag is turned off?

Shouldn't this be `if ($this->shouldCollect('models', true)) {`

Reply:
no